src/test/java/Constants.java

```diff
@@ -0,0 +1,32 @@
+public final class Constants {
```

Comment:
Separate class for constants is good 👍 

However, I don't think we need inner static classes here. For the future - it's better to create just a separate classes for them if you want (because if we'll have a LOT of constants for each class it will be hard to maintain one common class).

You can keep it 'as is' not though : )